src/main/java/step3/study/domain/Driver.java

```diff
@@ -11,19 +11,15 @@ public Driver(String name, Car car) {
         this.car = car;
     }
 
-    public void moveCar(int randomNumber) {
-        car.move(randomNumber);
+    public void moveCar(NumberGenerator numberGenerator) {
+        car.move(numberGenerator);
     }
 
-    public String getName() {
-        return name;
-    }
-
-    public String getDistance() {
-        return car.getDistance();
+    public int position() {
+        return car.position();
     }
 
-    public String getNameAndNowPosition() {
-        return this.name + " : " + car.getDistance();
+    public String name() {

```

Comment:
```suggestion
    public String getName() {
```

멤버는 명사로, 메서드는 명령형 동사를 사용해서 이름짓는게 좋습니다.